packages/employer-api/src/controllers/event.controller.ts

```diff
@@ -57,6 +58,16 @@ export const submission = async (req: any, res: express.Response) => {
                 )
                 if (addResult === 1) {
                     console.log("claim record update successful")
+                    // Send notifications to clients with Claims notifications enabled
+                    const { catchmentNo } = submission.data
```

Comment:
I believe here we are using the catchmentno from the form to determine who gets notified?

The catchmentno in the application submission will always be correct, but the catchmentno in the claim form will be wrong if the associated application was moved while the claim was in draft. I believe we need to use the authoritative catchmentno from the DB.

Reply:
Here I used a new request since the current code is comparing with 1, and did not want to break it since I cannot test locally due to CHEFS.